repair/row_level.cc

```diff
@@ -1456,15 +1479,26 @@ class repair_meta {
             if (rows.empty()) {
                 auto pk = dk_with_hash->dk.key();
                 last_dk_with_hash = dk_with_hash;
+                fragments_size = r.get_frozen_mutation().representation().size();
+                fragments_nr = 1;
                 rows.push_back(repair_row_on_wire(std::move(pk), {std::move(r.get_frozen_mutation())}));
+                rlogger.trace("New row to msg table={}.{} fragments_size={} fragments_nr={}", ks, cf, fragments_size, fragments_nr);
             } else {
                 auto& row = rows.back();
-                if (last_dk_with_hash && dk_with_hash->dk.tri_compare(*_schema, last_dk_with_hash->dk) == 0) {
+                // Avoid too many fragments in a single repair_row_on_wire which is sent as a rpc stream message
+                bool no_split = msg_split_feature ? fragments_size < max_fragments_size && fragments_nr < max_fragments_nr : true;
```

Comment:
What's the expected mf size? I wonder how much we can exceed the `max_fragments_size` and whether 
```
ragments_size + r.get_frozen_mutation().representation().size() < max_fragments_size
```
won't be safer

Reply:
Yes. Most conservative logic is split if (new mf would cause overflow) && !there_are_zero_mutation_fragments.